### ISSUE TITLE
Unittesting tasks that include logging with `Task.run` requires that I setup a context environment

### ISSUE THREAD

**Issue by dyllamt** [PrefectHQ/prefect#3211]
## Description
When I'm unittesting tasks, I manually call the `Task.run` method. Any task I have created that uses logging raises an error because there is no logger in the context (usually there would be a flow context).

## Expected Behavior
There should be a default logger in `prefect.context`, so calling `run` does not raise and `AttributeError`, and I don't have to set-up a context for each unittest.

## Reproduction
```
import prefect

@prefect.task
def log_task():
    logger = prefect.context.get("logger")
    logger.debug("In a flow, I want to do some logging")

log_task.run()  # AttributeError: 'NoneType' object has no attribute 'debug'
```

## Environment
{
    "platform": "macOS-10.15.5-x86_64-i386-64bit",
    "prefect_version": "0.13.3",
    "python_version": "3.8.3"
}

**Comment by dyllamt** [comment]
Here is the documentation on the recommended way to log the "decorator-style" tasks:
https://docs.prefect.io/core/concepts/logging.html#task-classes

**Comment by cicdw** [comment]
Hi @dyllamt - this issue was actually resolved w/ https://github.com/PrefectHQ/prefect/pull/3266 which was released with 0.13.6!
